src/ts/transformers/tab-helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { escapeHtml } from "../helpers/parse";
+import { KvTuple } from "./extract-details-keys";
+
+export function makeDefinitionList(dlKeyValues: KvTuple[], addClass: boolean = false) {
```

Comment:
originally from `html-details-body.ts`, it's not HAR specific so I've added it to it's own file.